lib/screens/profile/contact_support_public.dart

```diff
@@ -0,0 +1,292 @@
+  });
+
+  final TextEditingController nameController;
+  final TextEditingController emailController;
+  final TextEditingController messageController;
+  final Function(bool) onIncludeDeviceInfoChange;
+
+  @override
+  Widget build(BuildContext context) {
+    final design = DesignSystem.of(context);
+
+    final includeDeviceInfo = useState(true);
+
+    void includeDeviceInfoChange(bool state) {
+      includeDeviceInfo.value = state;
+      onIncludeDeviceInfoChange(state);
+    }
+
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        Padding(
+          padding: const EdgeInsets.all(16).copyWith(top: 0),
+          child: Text(
+            S.of(context).contactUs,
+            style: design.textStyles.headline1,
+          ),
+        ),
+        Expanded(
+          child: ListView(
+            padding: const EdgeInsets.all(16),
+            children: [
+              Container(
+                margin: const EdgeInsets.only(bottom: 10),
+                child: Text(
+                  S.of(context).contactSupportName,
+                  style: design.textStyles.caption1.copyWith(color: design.colors.label2),
+                ),
+              ),
+              Container(
+                margin: const EdgeInsets.only(bottom: 12),
+                child: TextField(
+                  cursorColor: design.colors.tint1,
+                  controller: nameController,
+                  style: design.textStyles.body2.copyWith(color: design.colors.label1),
+                  decoration: design.inputDecorations.textFormField.copyWith(
+                    hintText: S.of(context).contactSupportNameHint,
+                    suffixIcon: AnimatedSwitcher(
+                      duration: const Duration(milliseconds: 100),
+                      child: nameController.text.isEmpty
+                          ? null
+                          : GestureDetector(
+                              onTap: () => nameController.value = TextEditingValue.empty,
+                              child: Padding(
+                                padding: const EdgeInsets.only(right: 12),
+                                child: SvgPicture.string(
+                                  SvgIcons.clearXIcon,
+                                  theme: SvgTheme(currentColor: design.colors.label3),
+                                ),
+                              ),
+                            ),
+                    ),
+                  ),
+                ),
+              ),
+              Container(
+                margin: const EdgeInsets.only(bottom: 10),
+                child: Text(
+                  S.of(context).contactSupportEmail,
+                  style: design.textStyles.caption1.copyWith(color: design.colors.label2),
+                ),
+              ),
+              Container(
+                margin: const EdgeInsets.only(bottom: 12),
+                child: TextFormField(
```

Comment:
this is exactly the same as the signup email field, so that should be moved to another folder and reused instead